src/test/unit/auto_tag/test_auto_tag_event.py

```diff
@@ -0,0 +1,276 @@
+            "X-Amz-Cf-Id": ["J4KbOEUrZCnUQSLsDq1PyYXmfpVy8x634huSeBX0HCbscgH-N2AtVA=="],
+            "X-Amzn-Trace-Id": ["Root=1-5e9bf868-1c53a38cfe070266db0bfbd9"],
+            "X-Forwarded-For": ["5.102.206.161, 54.182.243.106"],
+            "X-Forwarded-Port": ["443"],
+            "X-Forwarded-Proto": ["https"],
+        },
+        "queryStringParameters": "1",
+        "multiValueQueryStringParameters": "1",
+        "pathParameters": "1",
+        "stageVariables": None,
+        "requestContext": {
+            "resourceId": "ua33sn",
+            "authorizer": {
+                "claims": {
+                    "sub": "a87005bb-3030-4962-bae8-48cd629ba20b",
+                    "custom:customer": "c_1111",
+                    "iss": "https://cognito-idp.us-west-2.amazonaws.com/us-west-2",
+                    "custom:customer-name": "a",
+                    "cognito:username": "aa",
+                    "aud": "4lidcnek50hi18996gadaop8j0",
+                    "event_id": "9fe80735-f265-41d5-a7ca-04b88c2a4a4c",
+                    "token_use": "id",
+                    "auth_time": "1587038744",
+                    "exp": "Sun Apr 19 08:06:14 UTC 2020",
+                    "custom:role": "admin",
+                    "iat": "Sun Apr 19 07:06:14 UTC 2020",
+                    "email": "a@a.com",
+                }
+            },
+            "resourcePath": "/add-user",
+            "httpMethod": "POST",
+            "extendedRequestId": "LOPAXFcuvHcFUKg=",
+            "requestTime": "19/Apr/2020:07:06:16 +0000",
+            "path": "/prod/add-user",
+            "accountId": "114300393969",
+            "protocol": "HTTP/1.1",
+            "stage": "prod",
+            "domainPrefix": "psqn7b0ev2",
+            "requestTimeEpoch": 1587279976628,
+            "requestId": "78542821-ca17-4e83-94ec-96993a9d451d",
+            "identity": {
+                "cognitoIdentityPoolId": None,
+                "accountId": None,
+                "cognitoIdentityId": None,
+                "caller": None,
+                "sourceIp": "5.102.206.161",
+                "principalOrgId": None,
+                "accessKey": None,
+                "cognitoAuthenticationType": None,
+                "cognitoAuthenticationProvider": None,
+                "userArn": None,
+                "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36",
+                "user": None,
+            },
+            "domainName": "psqn7b0ev2.execute-api.us-west-2.amazonaws.com",
+            "apiId": "psqn7b0ev2",
+        },
+        "body": '{"email":"a@a.com"}',
+        "isBase64Encoded": False,
+    }
+
+    AutoTagEvent.auto_tag_event(event=event)
+
+    assert SpansContainer.get_span().function_span[EXECUTION_TAGS_KEY] == [
+        {"key": "Accept", "value": "application/json, text/plain, */*"}
+    ]
```

Comment:
I think it will be nice to add a test with the tracer decorator. You can see examples in test_sync_hook.py